### DISCUSSION TITLE
Adjusting caching build to fix "Loading chunk failed" error in React

### DISCUSSION THREAD

**Discussion by peelar** [vercel/community#827] [Help] [answered]
Hello 👋 

I am having a pretty popular issue with invalidating cache on the deployment of a React application. The flow goes like this:
1. The user goes to the app and interacts with it
2. I redeploy the app in the meantime
3. The user clicks on a link and the browser is trying to load a chunk of JavaScript that doesn't exist anymore because the build cache was purged during the redeployment

Is there a way to alter the caching behavior on Vercel to make sure the build's JavaScript chunks stay available for some time after redeployment?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @peelar! You're right, this is a popular request! The Engineering team has this item on their list of feature requests and I'm adding your comment to that ticket for them to see. The cache is automatically invalidated with new deployments, so there is not currently a setting that will make the JS stay in cache after redeployment.
